sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSourceEventIds.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Diagnostics
+{
+    internal static class EventHubsEventSourceEventIds
```

Comment:
My concern with leaving these as a block of constants in the main source was trying to reduce clutter and avoid a giant region block that we rarely want to expand.  I didn't think of it until now, but we could also just mark the event source file as `partial` and define things as a big old block of constants in a dedicated `partial` file for it.

Reply:
I just saw your thought on "just pass it in as a parameter" - and I love that.